### DISCUSSION TITLE
first time user / launch

### DISCUSSION THREAD

**Discussion by JStead22** [philosowaffle/peloton-to-garmin#637] [Q&A]
I downloaded the 4.1.0 file, extracted it to desktop, ran the ClientUI.exe, I said run anyways after MS defender, then it spun and then nothing. 

![image](https://github.com/philosowaffle/peloton-to-garmin/assets/33033113/e2dfd8d1-5384-4073-babb-8730f5c3b387)

**Comment by jverke** [comment]
I also seem to be having this issue. I see the app running in the task bar, but there is no window or UI. I downloaded 4.3.1, and am running Windows 11.

**Reply by jverke** [reply]
I got the application to run--not sure if what I did was what made it work, but I figured I'd reply for science. I opened the launchSettings file in the Properties directory, and pasted one of the URLs in my browser, and tried to access it (unsuccessfully). After that, either by design or coincidence, I re-rean the exe, and it opened (and worked!) fine. Thank you for this great tool.

**Comment by philosowaffle** [comment]
Reasonably sure I've got this fixed in [v5](https://github.com/philosowaffle/peloton-to-garmin/releases/tag/v5.0.0).  There was an issue where P2G wouldn't launch on the very first try, but would launch successfully on follow up attempts.
